index.js

```diff
@@ -14,10 +25,12 @@ try {
     }
     region = 'eu-west-1';
 }
+const simpleDb = new SimpleDB({ region });
 
-const challongeService = challongeServiceFactory(challongeConfig);
-const userRepository = userRepositoryFactory({ region });
-const bot = botFactory(challongeService, userRepository);
+const challongeService =  challongeServiceFactory({ api, organization });
```

Comment:
What is the benefit of injecting via a single object parameter, instead of regular params -- `challongeServiceFactory(api, organization)` -- or each param curried separately -- `challongeServiceFactory(api)(organization)`?

Reply:
Hmm, I like to group deps and opts into objects if I know/suspect that I won't need to curry them. I'm not consistent with it tho...